<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_math</class>
 <widget class="QWidget" name="Form_math">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据统计</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ruijian_logo_v2.png</normaloff>ruijian_logo_v2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form_math{
    background-color: #FFFFFF;
    border-radius: 30px; 
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>68</y>
     <width>1328</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QTableWidget" name="tableWidget_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>631</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <column>
      <property name="text">
       <string>违禁品类型</string>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>具体类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最后检测时间</string>
      </property>
     </column>
    </widget>
    <widget class="PlotWidget" name="chart_1" native="true">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>691</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>0</y>
       <width>681</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border-radius:30px;
</string>
     </property>
     <property name="text">
      <string>每天违禁品数量趋势</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="chart_11">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>290</y>
       <width>691</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;

background-color: transparent;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>630</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <column>
      <property name="text">
       <string>违禁品类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>具体类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最后检测时间</string>
      </property>
     </column>
    </widget>
    <widget class="PlotWidget" name="chart_2" native="true">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>691</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>0</y>
       <width>681</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>每周违禁品数量趋势</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="chart_21">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>290</y>
       <width>691</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border: 1px solid rgb(75, 127, 232);
border-radius:30px;
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>630</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <column>
      <property name="text">
       <string>违禁品类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>具体类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最后检测时间</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>0</y>
       <width>681</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>月违禁品数量趋势</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="chart_3" native="true">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>691</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;
</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="chart_31">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>290</y>
       <width>691</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border: 1px solid rgb(75, 127, 232);
border-radius:30px;

</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>630</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;


</string>
     </property>
     <column>
      <property name="text">
       <string>违禁品类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>具体类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最后检测时间</string>
      </property>
     </column>
    </widget>
    <widget class="PlotWidget" name="chart_4" native="true">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>691</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>0</y>
       <width>691</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>每年违禁品数量趋势</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="chart_41">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>290</y>
       <width>691</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border: 1px solid rgb(75, 127, 232);
border-radius:30px;

</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>1241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(75, 127, 232);</string>
   </property>
   <property name="text">
    <string>本机违禁品统计</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	image: url(./image/z.png);
	border-radius:30px;
	border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>35</y>
     <width>1321</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;
color: rgb(75, 126, 232);</string>
      </property>
      <property name="text">
       <string>今日</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;
color: rgb(75, 126, 232);</string>
      </property>
      <property name="text">
       <string>近7天</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;
color: rgb(75, 126, 232);</string>
      </property>
      <property name="text">
       <string>近1个月</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(75, 127, 232);
border-radius:30px;
color: rgb(75, 126, 232);</string>
      </property>
      <property name="text">
       <string>近1年</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
